docs/03-tutorials/00-serverless/README.md

```diff
@@ -2,4 +2,70 @@
+
+# Use UI to explore
+
+At the beginning of your Kyma journey you will probably want to evaluate Serverless and draft a few Functions.
+Kyma Dashboard is perfect to gain basic experience and start the journey with Kyma Functions. Dashboard consists of UI components dedicated to Serverless that will help you draft your first Functions by putting the code directly in the browser using Web IDE.
+Kyma Dashboard will also help you expose your Function via HTTP, define environment variables, subscribe to Cloud Events, bind Service Instances and even show you the Function logs - all in single place.
+
+Get started with [Function UI](./svls-01-create-inline-function.md)
+
+![function-ui](./assets/svls-function-ui.png)
+
+# Use Kyma CLI for better development experience
+
+Defining Function from the UI is very quick and easy but most likely this approach would be not enough to satisfy your needs as a developer. The ability to code your Function from your favorite IDE or being able to run and debug the Function on your local machine before actually deploying in the Kyma runtime - Those aspects may be desired especially if you want to code and test a bit more complex cases. Also, you would probably want to avoid recreating same Functions manually from the UI on a different environment. In the end having deployable artefacts is more desirable. This is where Kyma CLI comes in handy as it allows you to keep your Function's code and configuration in a form of a workspace. 
+
+Initialize a scaffold for a brand new Function using the `kyma init function` command or fetch the current state of an existing Function deployed in your Kyma runtime using `kyma sync function`.
+Focus on the Function code and develop it from you favorite IDE. Configure your Functions directly in the [`config.yaml` manifest file](../../05-technical-reference/svls-06-function-configuration-file.md)
+>>NOTE: Use `kyma init function --vscode` to generate json schema which can be used in VScode for autocompletion.
+>>
+
+Kyma CLI helps you run your code locally with a single `kyma run function` command. It runs the Function using your local Docker daemon with the same runtime Docker context as if it was run in the Kyma runtime. 
+>>NOTE: Use `kyma run function` with `--hot-deploy` and spare yourself unnecessary restarts of the Functions whenever you test a changed Function logic. Also, use [`--debug` option](./svls-05-debug-function.md) to allow connecting with your favorite debugger.
+>>
+
+![kyma-cli-functions](./assets/svls-kyma-cli-functions.png)
+
+Having written and tested your Function locally, simply deploy it to the Kyma runtime with the `kyma apply function` command used in the folder of your Function's workspace. It will read the files, translate it to the Kubernetes manifests and deploy the Function.
```

Comment:
```suggestion
Having written and tested your Function locally, simply deploy it to the Kyma runtime with the `kyma apply function` command, used in the folder of your Function's workspace. It will read the files, translate it to the Kubernetes manifests and deploy the Function.
```